packages/pages/src/upgrade/upgrade.ts

```diff
@@ -2,12 +2,14 @@ import { Command } from "commander";
 import path from "path";
 import { updatePages } from "./pages_updater.js";
 import { migrateConfigs } from "./migrate_config.js";
+import { execSync } from "child_process";
 
 const handler = async () => {
-  // Pass CLI arguments as env variables to use in vite-plugin
   const rootPath = path.resolve("");
   await updatePages(rootPath);
   await migrateConfigs(rootPath);
+  execSync("pages generate templates");
```

Comment:
Hmm, it's probably more correct to call the functions rather than execute the shell script.